network/netplan/activate_test.go

```diff
@@ -120,7 +139,7 @@ func (s *ActivateSuite) TestActivateTimeout(c *gc.C) {
 			},
 		},
 		Directory: tempDir,
-		RunMode:   netplan.RunModeTimeout,
+		RunPrefix: "sleep 10000 && ",
```

Comment:
as a side thing, do we make sure that we don't have a 'sleep' processing running in the background after the test suite has run?